### ISSUE TITLE
Possibility to import old logs ?

### ISSUE THREAD

**Issue by jothoma1** [dbinoj/moodle-logstore_graylog#1]
Hi,
thanks a lot for this great plugin !!! 
It works great here, i just want to know if there is a possibility to import all logs with it ?

Thanks again :+1: 
Johan

PS: we are really interested into your TODO

> Use LearningLocker/Moodle-Log-Expander or something similar to expand IDs. Better if logstore evolves automatically whenever the Moodle-Log-Expander project has new log types implemented.

**Comment by dbinoj** [comment]
Hello!
Thanks for the feedback! And regarding your question, yes its possible. I imported ~3 years worth of logs into graylog. Just select "Background"  mode. The plugin fetches 100000 log entries per scheduled task run until it ships the entire log table.

Regarding the TODO, I needed a solution before I got a chance to implement a clean solution in the module itself. So ended up doing all aggregations & reporting in graylog and populated the IDs in the reports by querying Moodle's DB. So, that is not on my priority list, unfortunately. Will do look into it during vacations :+1:
